src/utils/calendar-access.js

```
@@ -0,0 +1,122 @@
+        
+        // エラーチェック
+        if (result && result.error) {
+          const toolError = new Error(result.error);
+          toolError.isToolError = true;
+          reject(toolError);
+          return;
+        }
+        
+        resolve(result);
+      } catch (parseError) {
+        console.error('カレンダーツール出力のパースに失敗:', parseError);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)